### PR TITLE
Vitalii Bondarenko: on review

### PR DESCRIPTION
**Pull request by VitaliyBondarenko1982**
[DEMO LINK](https://VitaliyBondarenko1982.github.io/react_movies-list-add-form/)

### CODE REVIEW

File: src/components/NewMovie/NewMovie.scss [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    top: 50px;
+    opacity: 0;
+    &--visible {
+      color: red;
+      opacity: 1;
+    }
+  }
+
+  &__button {
+    // background-color: lightgrey;
```
Comment (irenhh) [inline src/components/NewMovie/NewMovie.scss]: remove this

File: src/components/NewMovie/NewMovie.js [hunk 2]
```diff
@@ -1,13 +1,188 @@
...
+    const { name, value } = target;
+
+    this.setState({
+      [name]: value,
+      imgUrlError: false,
+      imdbUrlError: false,
+    });
+  }
+
+  submitHandler = (evt) => {
```
Comment (irenhh) [inline src/components/NewMovie/NewMovie.js]: one setState per method

File: src/components/NewMovie/NewMovie.js [hunk 3]
```diff
@@ -1,13 +1,188 @@
...
+    evt.preventDefault();
```
Comment (irenhh) [inline src/components/NewMovie/NewMovie.js]: event

File: src/components/NewMovie/NewMovie.js [hunk 4]
```diff
@@ -1,13 +1,188 @@
...
+      description,
+      imgUrl,
+      imdbUrl,
+      imdbId,
+      imgUrlError,
+      imdbUrlError,
+    } = this.state;
+
+    const imgUrlClassName = (imgUrlError)
+      ? 'form__error--visible' : 'form__error';
```
Comment (irenhh) [inline src/components/NewMovie/NewMovie.js]: use classnames lib
https://www.npmjs.com/package/classnames

**Comment by irenhh** [comment]
you didn't fix the very first comment

**Comment by VitaliyBondarenko1982** [comment]
> you didn't fix the very first comment

Забув зробити deploy))

**Pull request closed without merging**